src/Components/Referrer/referrerHook.tsx

```diff
@@ -18,6 +18,8 @@ export type ReferrerData = {
   shareLink: ReferrerOptions<string>;
   featureFlags: ReferrerOptions<string[]>;
   stepDirectory: ReferrerOptions<StepDirectory>;
+  noResultMessage: ReferrerOptions<{ id: string; defaultMessage: string }>;
```

Comment:
If you put `{"_label": "someLabel", "_default_message": "some message"}` in the config. It will automatically be converted into a `FormattedMessage` component that you can just use in your JSX.
```suggestion
  noResultMessage: ReferrerOptions<FormattedMessageType>;
```

Reply:
done